paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetActivityTest.kt

```diff
@@ -584,16 +584,10 @@ internal class PaymentSheetActivityTest {
 
             composeTestRule.waitForIdle()
 
-            testDispatcher.scheduler.apply {
-                advanceTimeBy(PrimaryButtonAnimator.HOLD_ANIMATION_ON_SLIDE_IN_COMPLETION)
-                runCurrent()
-            }
-
             assertThat(finishProcessingCalled).isTrue()
         }
     }
```

Comment:
I've unfortunately had to disable this test because we have logic we do not execute in `PrimaryButton` when in inspection mode. Should we look into creating separate modes for all our testing & inspection strategies?
- `LocalInspectionMode`
- `LocalScreenshotMode`
- `LocalTestingMode`